examples/step-8/step-8.cc

```diff
@@ -648,37 +649,7 @@ namespace Step8
 // step-6 (apart from the changed class names, of course).
 int main()
 {
-  try
-    {
-      Step8::ElasticProblem<2> elastic_problem_2d;
-      elastic_problem_2d.run();
-    }
-  catch (std::exception &exc)
-    {
-      std::cerr << std::endl
-                << std::endl
-                << "----------------------------------------------------"
-                << std::endl;
-      std::cerr << "Exception on processing: " << std::endl
-                << exc.what() << std::endl
-                << "Aborting!" << std::endl
-                << "----------------------------------------------------"
-                << std::endl;
-
-      return 1;
-    }
-  catch (...)
-    {
-      std::cerr << std::endl
-                << std::endl
-                << "----------------------------------------------------"
-                << std::endl;
-      std::cerr << "Unknown exception!" << std::endl
-                << "Aborting!" << std::endl
-                << "----------------------------------------------------"
-                << std::endl;
-      return 1;
-    }
-
+  Step8::ElasticProblem<2> elastic_problem_2d;
+  elastic_problem_2d.run();
```

Comment:
From #6784: we should keep the handler so that MSVC can print an error message.

Reply:
done.